### PR TITLE
auditfix: veto: support cancelling txs on both timelockV1 and current timelock

### PR DESCRIPTION
**Pull request by davidbrai**
fixes: https://github.com/spearbit-audits/review-nouns/issues/7

### CODE REVIEW

File: packages/nouns-contracts/contracts/governance/NounsDAOV3Proposals.sol [hunk 1]
```diff
@@ -540,8 +540,9 @@ library NounsDAOV3Proposals {
         NounsDAOStorageV3.Proposal storage proposal = ds._proposals[proposalId];
 
         proposal.vetoed = true;
+        INounsDAOExecutor timelock = getProposalTimelock(ds, proposal);
```
Comment (eladmallel) [inline packages/nouns-contracts/contracts/governance/NounsDAOV3Proposals.sol:543]: not critical, but a nice practice to add unit tests in our fix PRs, wdyt?

Comment (davidbrai) [reply packages/nouns-contracts/contracts/governance/NounsDAOV3Proposals.sol:543]: @eladmallel generally yes, but in this case we don't have a good testing setup that I can easily duplicate

Comment (davidbrai) [reply packages/nouns-contracts/contracts/governance/NounsDAOV3Proposals.sol:543]: @eladmallel do you think it's ok to merge?

**Comment by eladmallel** [comment]
@davidbrai I'm ok merging without a test

**Pull request merged**